packages/agent/src/cbor.test.ts

```diff
@@ -54,6 +54,85 @@ test('empty canister ID', () => {
   expect(Principal.fromUint8Array(outputA as any).toText()).toBe('aaaaa-aa');
 });
 
+describe('cbor encode/decode bigint', () => {
+  test('encode decode 0 to 1_000_000', () => {
+    for (let i = 0; i < 1_000_000; i += 1) {
+      let buffer;
+
+      try {
+        buffer = encode(BigInt(i));
+      } catch (error) {
+        console.log(`error encoding on ${i.toString()}`);
+        expect((error as Error).message).toBeTruthy();
+      }
+
+      try {
+        const decoded = decode<bigint>(buffer);
+        expect(decoded).toBe(BigInt(i));
+      } catch (error) {
+        console.log(`error decoding on ${i.toString()}`);
+        expect((error as Error).message).toBeTruthy();
+      }
+      // // if we log the time
+      // if (i % 100000 === 0) {
+      //   console.log(`${new Date(Date.now()).toLocaleTimeString()} with ${i.toString()}`);
```

Comment:
Is there a reason these comments are left in?

Reply:
Can delete those